beacon-chain/state/fieldtrie/field_trie_helpers.go

```diff
@@ -148,34 +138,6 @@ func convertBalances(indices []uint64, elements interface{}, convertAll bool) ([
 	return handleBalanceSlice(val, indices, convertAll)
 }
 
-// handleByteArrays computes and returns byte arrays in a slice of root format.
-func handleByteArrays(val [][]byte, indices []uint64, convertAll bool) ([][32]byte, error) {
-	length := len(indices)
-	if convertAll {
-		length = len(val)
-	}
-	roots := make([][32]byte, 0, length)
-	rootCreator := func(input []byte) {
-		newRoot := bytesutil.ToBytes32(input)
```

Comment:
This assumes that the roots are obtained by packing on the right "BigEndian style". Probably better to document this in the function

Reply:
I am removing this function all together. If this comment is about `handle32ByteArrays`, then I'm happy to change in a later PR because this one didn't touch the function